src/api/common_api.js

```diff
@@ -252,6 +252,15 @@ module.exports = {
             }
         },
 
+        bucket_policy_condition: {
+            type: 'object',
+            properties: {
+                serverSideEncryption: {
```

Comment:
Stile: 
this is a camelcase style, we are using snake_case
```suggestion
                server_side_encryption: {
```